appengine/src/main/java/com/google/googleinterns/gscribe/GScribeApplication.java

```diff
@@ -32,17 +38,21 @@ public static void main(String[] args) throws Exception {
 
   @Override
   public String getName() {
-    return "HelloWorld";
+    return "GScribeApplication";
   }
 
   @Override
   public void initialize(Bootstrap<GScribeConfiguration> bootstrap) {
     bootstrap.setConfigurationSourceProvider(new ResourceConfigurationSourceProvider());
-    bootstrap.addBundle(new AssetsBundle("/assets", "/", "index.html" ));
+    bootstrap.addBundle(new AssetsBundle("/assets", "/", "index.html"));
   }
 
   public void run(GScribeConfiguration configuration, Environment environment) throws Exception {
+    Injector injector = Guice
+        .createInjector(new ConfigModule(com.google.googleinterns.gscribe.Environment.LOCAL),
```

Comment:
you can derive the environment from an environment variable passed to the java command while starting the app.
Please add a TODO here.